src/scripts/components/menu/menu.tsx

```diff
@@ -8,67 +8,132 @@ import {LanguageIcon} from '../icons/language-icon';
-        </li>
-        <li className={styles.menuListItem}>
-          <Button
-            className={styles.menuButton}
-            label={'presenterMode'}
-            link={'/present'}
-          />
-        </li>
-        <li className={styles.menuListItem}>
-          <Button
-            className={styles.menuButton}
-            label={'showcaseMode'}
-            link={'/showcase'}
-          />
-        </li>
-        <li className={styles.subMenuTitle}>
-          <DownloadIcon /> <FormattedMessage id={'offline'} />
-        </li>
-        <li className={styles.menuListItem}>
-          <Button className={styles.menuButton} label={'download'} />
-        </li>
-      </ul>
-      <ul className={styles.menuList}>
-        <li className={styles.menuItemTitle}>
-          <LanguageIcon /> <FormattedMessage id={'language'} />
-        </li>
-        <li>
-          <LanguageSelector className={styles.menuListItem} />
-        </li>
-      </ul>
-      <ul className={styles.menuList}>
-        <li className={styles.menuItemTitle}>
-          <InfoIcon /> <FormattedMessage id={'info'} />
-        </li>
-        <li className={styles.menuListItem}>
-          <Button className={styles.menuButton} label={'about'} />
-        </li>
-        <li className={styles.menuListItem}>
-          <Button className={styles.menuButton} label={'EsaWebsite'} />
-        </li>
-        <li className={styles.menuListItem}>
-          <Button className={styles.menuButton} label={'CCIWebsite'} />
-        </li>
-        <li className={styles.menuListItem}>
-          <Button className={styles.menuButton} label={'github'} />
-        </li>
-      </ul>
-    </nav>
-    <div className={styles.logo}>
-      <CCILogo />
-    </div>
-  </React.Fragment>
-);
+const Menu: FunctionComponent = () => {
+  // @ts-ignore - injected via webpack's define plugin
+  const version = INFO_VERSION;
+  const [showOverlay, setShowOverlay] = useState(false);
+
+  return (
+    <React.Fragment>
+      <nav className={styles.menuContainer}>
+        <ul className={styles.menuList}>
+          <li className={styles.menuItemTitle}>
+            <PresenterIcon /> <FormattedMessage id={'modes'} />
+          </li>
+          <li className={styles.menuListItem}>
+            <Button
+              className={styles.menuButton}
+              label={'presenterMode'}
+              link={'/present'}
+            />
+          </li>
+          <li className={styles.menuListItem}>
+            <Button
+              className={styles.menuButton}
+              label={'showcaseMode'}
+              link={'/showcase'}
+            />
+          </li>
+          <li className={styles.subMenuTitle}>
+            <DownloadIcon /> <FormattedMessage id={'offline'} />
+          </li>
+          <li className={styles.menuListItem}>
+            <a
+              href={`https://storage.googleapis.com/esa-cfs-versions/electron/${version}/esa-climate-from-space-${version}-win.exe`}
```

Comment:
Would be great if the urls come from the main config file similar to the other urls like `baseUrlTiles` etc. You can insert the version there already.